quiche/src/recovery/gcongestion/bbr/bandwidth_sampler.rs

```diff
@@ -915,11 +917,11 @@ mod bandwidth_sampler_tests {
                 self.round_trip_count,
             );
 
-            let max_bandwidth =
-                self.max_bandwidth.max(sample.sample_max_bandwidth.unwrap());
+            let sample_max_bandwidth = sample.sample_max_bandwidth.unwrap();
+            self.max_bandwidth = self.max_bandwidth.max(sample_max_bandwidth);
 
             let bandwidth_sample = BandwidthSample {
-                bandwidth: max_bandwidth,
+                bandwidth: sample_max_bandwidth,
```

Comment:
The original test does not do `self.max_bandwidth.max` on the bw sample returned by this function in the test harness.  This fixes the `app_limited` test at idx `61`.

https://github.com/google/quiche/blob/66dc426afd733258dd95b6e3307a5833a48e5df4/quiche/quic/core/congestion_control/bandwidth_sampler_test.cc#L102